index.html

```diff
@@ -10,7 +10,12 @@
 </head>
 <body>
 
-    <h1>Hello world!!!</h1>
+    <div class="container">
+        <p> ime grada</p>
+        <input type="search" name="" class="searchbar">
```

Comment:
delete the atribute name and add placeholder="Please enter your place here"

Reply:
I make changes